application/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ endif()
 add_executable(f3d ${F3D_SOURCE_FILES})
 target_link_libraries(f3d PUBLIC libf3d)
 
+if(UNIX)
+  target_link_libraries(f3d PRIVATE pthread)
```

Comment:
I think the recommended method is:

```suggestion
  find_package(Threads REQUIRED)
  target_link_libraries(f3d PRIVATE Threads::Threads)
```

Reply:
https://stackoverflow.com/questions/5395309/how-do-i-force-cmake-to-include-pthread-option-during-compilation

indeed